File: sdk-py/examples/train.py
import boxty as bx

client = bx.Client(api_key="bxty_...")
sandbox = client.create_sandbox(image="pytorch:latest", cpu=4, memory=16384, gpu="A100")
result = sandbox.exec("python train.py --epochs=10")
print(result.stdout)
sandbox.stop()

File: sdk-js/examples/train.js
import { Client } from 'boxty';

const client = new Client({ apiKey: 'bxty_...' });
const sandbox = await client.createSandbox({ image: 'pytorch:latest', cpu: 4, memory: 16384, gpu: 'A100' });
const result = await sandbox.exec('python train.py --epochs=10');
console.log(result.stdout);
await sandbox.stop();
